trace_decoder/Cargo.toml

```diff
@@ -20,6 +20,7 @@ hex = { workspace = true }
 hex-literal = { workspace = true }
 keccak-hash = { workspace = true }
 log = { workspace = true }
+plonky2 = "0.2.1"
```

Comment:
Is there a reason we're not on 0.2.2?

Reply:
Nah. I think this might just be from basing the branch off of an older branch.